contracts/BC_fusion/SystemConfig.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+contract SystemConfig {
```

Comment:
Hope we can still reuse the global `System.sol`

Reply:
I c.  I think we need simplify the `system.sol`, a lot of unused code.

Reply:
Done